modules/administration/pages/tuning-largedeploys.adoc

```diff
@@ -37,9 +32,9 @@ Size of updates * Number of clients / Theoretical download speed / 60
 ----
 
 * Depending on hardware you can accept hundreds of client keys.
-* Plan time for onboarding clients{mdash} at least one hour per 1000 clients.
+* Plan time for onboarding clients{mdash}at least one hour per 1000 clients.
 * It is not recommended onboarding more than approx.
-1000 clients directly to the {productname} server{mdash} proxies should be used instead.
+1000 clients directly to the {productname} server{mdash}proxies should be used instead.
```

Comment:
Personally, I would add a comma to 1,000 but Chicago is ambivalent on it, so it's up to you.

Reply:
I prefer to add a comma if there are 5 digits.  But if we have a majority for 4, also fine for me.

Reply:
Sure, I'm not fussed either way :)